import.sh

```diff
@@ -60,9 +60,15 @@ import() {
 	esac
 
 	# The base directory for the import cache.
-	# Defaults to `$HOME/.import_cache`.
+	# Defaults to `import.pw` in the user cache directory specified by `$XDG_CACHE_HOME`
+	# or `$LOCALAPPDATA` (falling back to `$HOME/Library/Caches` on macOS and
+	# `$HOME/.cache` everywhere else).
 	# May be configured by setting the `IMPORT_CACHE` variable.
-	local cache="${IMPORT_CACHE-${HOME}/.import-cache}"
+	# On AWS Lambda, `$HOME` is not defined but `~` works.
+	# Furthermore, make sure we can always set IMPORT_CACHE even if HOME is undefined.
+	local ucd_fallback="${HOME:-"$(echo ~)"}/.cache"
+	[ "$(uname -s)" = "Darwin" ] && ucd_fallback="${HOME:-}/Library/Caches"
```

Comment:
The macOS line needs the fallback as well:

```suggestion
	local home="${HOME:-"$(echo ~)"}"
	local ucd_fallback="$home/.cache"
	[ "$(uname -s)" = "Darwin" ] && ucd_fallback="$home/Library/Caches"
```

Reply:
updated